apps/engmanagement/src/pages/[slug].tsx

```diff
@@ -181,9 +182,20 @@ const allPostsQuery = groq`
   }
 `
 
+function getConvertkitFromCookieHeaders(serverCookies: string = '') {
+  return CK_SUBSCRIBER_KEY
+    ? serverCookie.parse(serverCookies)[CK_SUBSCRIBER_KEY]
+    : ''
+}
+
 export const getServerSideProps: GetServerSideProps = async (context: any) => {
   const allPosts = await sanityClient.fetch(allPostsQuery)
   const currentPost = find(allPosts, {slug: context.params.slug})
+  const cookieHeader = context.req.headers.cookie as string
+
+  const convertkitId =
+    context.query[CK_SUBSCRIBER_KEY] ||
+    getConvertkitFromCookieHeaders(cookieHeader)
```

Comment:
the id can be in a query or the cookie at this point so we can check both real quick and get an id